iceoryx_hoofs/include/iceoryx_hoofs/cxx/optional.hpp

```diff
@@ -296,6 +296,17 @@ bool operator!=(const nullopt_t, const optional<T>& rhs) noexcept;
 /// @return true if the optional is set, otherwise false
 template <typename T>
 bool operator!=(const optional<T>& lhs, const nullopt_t) noexcept;
+
+template <typename... T>
+struct is_optional : std::false_type
+{
+};
+// AXIVION Next Construct AutosarC++19_03-A14.7.2 : We are extending std, also, it is allowed as
+// specialization for user-defined type.
```

Comment:
Shouldn't this warning disappear since the everything is now in the same file?

Reply:
@elBoberido corrected.